express/blocks/billing-radio/billing-radio.js

```diff
@@ -0,0 +1,86 @@
+// fires 'billing-plan' BM and has global sync values when multiple on same page
```

Comment:
What's happening here with the commented out code?

Reply:
This is just copied over from express

Reply:
@fullcolorcoder - Technically the Billing Radio block is deprecated, however, there may be some situations where it is authored in a doc. The point off this code is to just remove the block if it added in a doc.